delta/plugins/storage/src/test/resources/storage/events/storage-created.json

```diff
@@ -0,0 +1,23 @@
+{
+  "@context" : [
+    "https://bluebrain.github.io/nexus/contexts/metadata.json",
+    "https://bluebrain.github.io/nexus/contexts/storages.json"
+  ],
+  "_storageId" : "https://bluebrain.github.io/nexus/vocabulary/s3-storage",
+  "_project" : "org/project",
+  "_source" : {
+    "secretKey" : "CtDXqfq6P2/FdfD+TwvxPw==",
```

Comment:
I'm wondering if we should replace these values with `"****"`. Thoughts?
When we fetch an `S3Storage` resource, do we return the unencrypted values?

Reply:
On 1.4.x we return the encrypted values on the SSEs.

Reply:
ok, great!

Reply:
But If you want we can remove them. To have a field with `****` is useless. It would be either to provide them encrypted or not at all.

Reply:
if the values are encrypted and that's the v1.4 behaviour, then let's keep this as is.